Installation/CHANGES.md

```diff
@@ -6,6 +6,9 @@ Release History
 
 Release date: June 2020
 
+### Add Move Semantics to AABB Trees
+ -   Added move constructor and assignment operator to AABB Tree class.
+
```

Comment:
Should be move to the new section about CGAL-5.2, after a merge of `master`.

Reply:
@theartful Since I'm responsible for the conflicts, let me try to do the merge and I'll let you verify I did not do anything wrong.